src/helpers/__tests__/replaceWithCurrentDomain.test.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import { replaceDomain } from '../replaceWithCurrentDomain';
+
+const testCases = [
+  ['https://example.com', 'https://domain.net', 'https://domain.net/'],
+  ['https://example.com', 'https://domain2.org', 'https://domain2.org/'],
+  ['https://kiwi.com/asdg/4/', 'https://kiwi.org', 'https://kiwi.org/asdg/4/'],
+];
+
+describe('replaceDomain', () => {
+  testCases.forEach(([URL, domain, result]) => {
```

Comment:
This test seems to be unnecessary complicated to comprehend, just passing one arg and checking the result is fine